NEWS

```diff
@@ -1,3 +1,36 @@
-Changes in version 0.99.0 (2021-09-29)
+# Changes (2022-05-30)
+
+## added `names_species` and `names_resources` in related functions
```

Comment:
This is marked as a subtitle ("##" in markdown). I assume this should be normal text? The subtitle could be 

## New arguments

-> Then also specify which related functions.